### PR TITLE
iris1.7.4, cartopy12

### PR DESCRIPTION
**Pull request by marqh**

### CODE REVIEW

File: cartopy/meta.yaml [hunk 1]
```diff
@@ -39,8 +38,7 @@ requirements:
     - owslib
     - numpy
     - proj4
-    - shapely ==1.4.3.issue177  # [osx]
-    - shapely                   # [not osx]
+    - shapely ==1.5.6
```
Comment (pelson) [inline cartopy/meta.yaml]: There are good reasons for the pinned version of shapely on OSX:

https://github.com/Toblerity/Shapely/issues/177

Comment (pelson) [inline cartopy/meta.yaml]: Also, shapely is part of this repository: https://github.com/SciTools/conda-recipes-scitools/tree/master/shapely

File: iris/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    version: 1.7.4
 
 source:
     git_url: https://github.com/SciTools/iris.git
-    git_tag: v1.7.x
+    git_tag: v1.7.4
 
 build:
-    number: 2  # [win]
+    number: 0  # [win]
```
Comment (pelson) [inline iris/meta.yaml]: Just remove the Windows condition?

**Comment by pelson** [comment]
On OSX
----------

```
Assertion failed: (0 != cs), function GEOSCoordSeq_getSize_r, file geos_ts_c.cpp, line 3612.
```

This is because of GEOS being different between build and run. In our case, this is because shapely has different build and run GEOS determination, and is what the patched version fixed.

On Windows
----------------
```
conda.resolve.NoPackagesFound: No packages found in current win-32 channels matching: shapely ==1.5.6 
Command exited with code 1
```

This is because there isn't yet a Shapely for windows at this version.


For what it is worth, if we manage to fix the OSX problem at the shapely level, I would cease to pin specific Shapely versions (perhaps for cartopy I would state >=1.5.5 or whatever it is).

**Comment by bjlittle** [comment]
:+1:

**Pull request merged**